src/git/util.js

```diff
@@ -23,7 +29,8 @@ const defaultBranchTemplate = '{issue.shortName}/{issue.key}_{issue.sanitizedSum
 
 const getTemplateData = (config, issue) => ({
```

Comment:
Now that the template needs to be accessed in two places, I would split this function in two:

* `getTemplate` returning just the template string.
* `getTemplateData` returning what now is on data.